rust/cubeclient/src/apis/default_api.rs

```diff
@@ -4,19 +4,21 @@ use uuid::Uuid;
 
 use super::{configuration, Error};
 use crate::apis::ResponseContent;
+use crate::models::{V1LoadResponse400, V1MetaResponse500};
 
 /// struct for typed errors of method `load_v1`
 #[derive(Debug, Clone, Serialize, Deserialize)]
 #[serde(untagged)]
 pub enum LoadV1Error {
-    Status400(crate::models::V1LoadResponse400),
+    Status400(V1LoadResponse400),
     UnknownValue(serde_json::Value),
 }
 
 /// struct for typed errors of method `meta_v1`
 #[derive(Debug, Clone, Serialize, Deserialize)]
 #[serde(untagged)]
 pub enum MetaV1Error {
+    Status500(V1MetaResponse500),
```

Comment:
https://github.com/cube-js/cube.js/blob/master/packages/cubejs-api-gateway/openspec.yml#L154

Maybe we need to rename & change this structure to be a common case for all errors

Reply:
https://github.com/cube-js/cube.js/pull/3835